cmd/oci-create-runtime-bundle/main.go

```diff
@@ -37,30 +37,25 @@ var bundleTypes = []string{
 }
 
 type bundleCmd struct {
-	stdout  *log.Logger
-	stderr  *log.Logger
+	context.Context
 	typ     string // the type to bundle, can be empty string
 	ref     string
 	root    string
+	debug   bool
 	version bool
 }
 
 func main() {
-	stdout := log.New(os.Stdout, "", 0)
-	stderr := log.New(os.Stderr, "", 0)
-
-	cmd := newBundleCmd(stdout, stderr)
+	ctx := context.Background()
+	ctx = context.WithLogger(ctx, context.G(ctx))
```

Comment:
@vbatts You are right. It is updated to https://github.com/opencontainers/image-tools/pull/46/commits/548a4ae124f9e9b6339f3b262223a62ceb18aef0 and https://github.com/opencontainers/image-tools/pull/46/commits/eb76ffbfbd777a6797c476ec4abcdb0390ed51b6

The changeset is:
https://github.com/opencontainers/image-tools/pull/46/commits/eb76ffbfbd777a6797c476ec4abcdb0390ed51b6#diff-bd4694bbf0ec2644693b1ef195e3d330R49
https://github.com/opencontainers/image-tools/pull/46/commits/eb76ffbfbd777a6797c476ec4abcdb0390ed51b6#diff-8c24ca9b63b3598ef3f4cc64741f433eR54
https://github.com/opencontainers/image-tools/pull/46/commits/eb76ffbfbd777a6797c476ec4abcdb0390ed51b6#diff-cfb3ced180edd25cfc0a028c1a6e900dR48

Changed `logentry` to `LogEntry` as public variable. And then in `*/main.go` it is applied as,
```go
ctx := context.WithLogger(context.Background(), LogEntry)
```
That improves declaration of `context` to oneline, and keep context resource usage unchanged.